src/test/java/com/wikia/webdriver/elements/mercury/components/Search.java

```diff
@@ -27,7 +27,7 @@
   public static final int FOCUS_TIMEOUT_IN_SECONDS = 1;
   public static final int SUGGESTIONS_TIMEOUT_IN_SECONDS = 1;
 
-  private static final String searchSuggestionClass = ".wikia-search__suggestions li.mw-content a";
+  private static final String searchSuggestionClass = ".wikia-search__suggestions li.mw-content";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)